Update UI for `SitterListView` to present information about loading status and errors if needed. (#218)
ios-course/ironfoudation-team-project: CHANGELOG.md

 - Update PR template [#194](https://github.com/ios-course/ironfoudation-team-project/pull/194)
 - Update revision of the `SimpleNetworkService` package in the project - [#204](https://github.com/ios-course/ironfoudation-team-project/pull/204)
 - Update `SitterProfileView` to allow the user to enter profile info only once - [#212](https://github.com/ios-course/ironfoudation-team-project/pull/212)

>  - Update `SitterListView` to present information about loading status and errors if needed - [#218](https://github.com/ios-course/ironfoudation-team-project/pull/218)

Reviewer: nit: 
```suggestion
 - Update `SitterListView` to present a loading indicator and occurring errors - [#218](https://github.com/ios-course/ironfoudation-team-project/pull/218)
```

Author: Fixed
